schema.sql

```diff
@@ -22,3 +22,14 @@ postgres=# \d vet_clinic
 -- Add a column species of type string to your animals table. Modify your schema.sql file.
 postgres=# ALTER TABLE animals
 postgres-# ADD species varchar(100);
+
+
+-- Third task
+
+postgres=# CREATE TABLE owners (id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY, full_name VARCHAR(120), age INT);
+
+postgres=# CREATE TABLE species (id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY, name VARCHAR(120));
+
+postgres=# ALTER TABLE animals DROP COLUMN species;
+
+postgres=# ALTER TABLE animals ADD COLUMN species_id INT,ADD COLUMN owner_id INT,ADD FOREIGN KEY (species_id) REFERENCES species(id), ADD FOREIGN KEY (owner_id) REFERENCES owners(id);
```

Comment:
- [ ] Great job so far 👏🏻 however According to the project requirements, you should take a screenshot of the results of your queries. Then add the screenshot with the results of your queries to your Pull Request description. Kindly address this ✍🏻

Reply:
@BenMukebo I did add it to the folder in thirdimage.

Reply:
@Olamarx!!  Kindly add the screenshot with the results to your Pull Request

Reply:
The requirement does not state adding the screenshot of creating data, sir.

Reply:
Hi @Olamarx 
I suggest you read all your **[project's requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md)** as everything is specified there.